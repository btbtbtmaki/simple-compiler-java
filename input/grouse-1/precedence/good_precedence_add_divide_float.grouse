main {

  print 1.7 + 3.1 / 3.8 nl;
  print 8.6 + 2.6 / 9.2 nl;
  print 5.1 + 7.9 / 5.1 nl;
  print 2.2 + 7.7 / 9.1 nl;
  print 9.0 + 4.3 / 1.0 nl;
  print 3.9 + 2.7 / 2.4 nl;
  print 1.9 + 3.0 / 9.7 nl;
  print 5.1 + 6.9 / 6.5 nl;
  print 6.5 + 1.2 / 3.0 nl;
  print 9.3 + 6.0 / 9.7 nl;
  print 3.7 + 4.6 / 6.3 nl;
  print 2.9 + 3.5 / 7.0 nl;

  print 9.8 / 4.5 + 1.1 nl;
  print 8.5 / 6.1 + 8.6 nl;
  print 6.8 / 9.3 + 8.2 nl;
  print 4.4 / 3.2 + 2.2 nl;
  print 6.5 / 1.3 + 1.6 nl;
  print 8.6 / 9.1 + 4.0 nl;
  print 1.1 / 7.9 + 8.2 nl;
  print 3.8 / 6.4 + 6.7 nl;
  print 7.4 / 1.9 + 2.8 nl;
  print 1.8 / 6.5 + 7.5 nl;
  print 6.9 / 1.3 + 2.5 nl;
  print 5.1 / 1.2 + 4.0 nl;

  print 1.7 + 1.3 / 2.5 + 4.8 nl;
  print 5.5 + 3.1 / 6.9 + 2.4 nl;
  print 2.3 + 9.0 / 5.7 + 4.6 nl;
  print 9.6 + 4.1 / 8.5 + 2.6 nl;
  print 1.1 + 9.2 / 7.4 + 2.8 nl;
  print 7.7 + 5.6 / 1.5 + 1.2 nl;
  print 7.4 + 3.5 / 3.7 + 4.9 nl;
  print 9.6 + 6.7 / 4.6 + 3.5 nl;
  print 6.7 + 3.0 / 8.1 + 8.7 nl;
  print 2.2 + 9.3 / 1.0 + 9.0 nl;
  print 8.4 + 1.4 / 9.2 + 6.7 nl;
  print 9.9 + 2.6 / 2.6 + 7.5 nl;

  print 8.4 / 7.4 + 9.6 / 8.5 nl;
  print 4.1 / 1.6 + 3.0 / 1.2 nl;
  print 3.1 / 5.0 + 6.2 / 8.6 nl;
  print 6.7 / 3.8 + 9.6 / 6.2 nl;
  print 2.5 / 8.4 + 1.1 / 1.0 nl;
  print 5.8 / 8.5 + 8.9 / 1.8 nl;
  print 1.2 / 7.0 + 2.2 / 6.0 nl;
  print 4.1 / 7.6 + 4.8 / 7.1 nl;
  print 6.5 / 8.1 + 2.9 / 5.0 nl;
  print 9.1 / 3.8 + 2.3 / 1.7 nl;
  print 3.9 / 4.4 + 6.7 / 4.4 nl;
  print 1.8 / 4.6 + 6.9 / 1.0 nl;

}

