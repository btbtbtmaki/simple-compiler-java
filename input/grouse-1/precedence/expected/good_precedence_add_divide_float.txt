2.51579
8.88261
6.64902
3.04615
13.3
5.025
2.20928
6.16154
6.9
9.91856
4.43016
3.4
3.27778
9.99344
8.93118
3.575
6.6
4.94505
8.33924
7.29375
6.69474
7.77692
7.80769
8.25
7.02
8.34928
8.47895
12.6824
5.14324
12.6333
13.2459
14.5565
15.7704
20.5
15.2522
18.4
2.26455
5.0625
1.34093
3.31154
1.39762
5.6268
0.538095
1.21553
1.38247
3.74768
2.40909
7.2913
